app/components/media/prev_next_component.html.erb

```diff
@@ -1,4 +1,4 @@
-<div class="prev-next-container">
+<div class="prev-next-container" role="navigation">
```

Comment:
Can we just make this a `<nav>` node instead?  Either way, I think this needs an aria-label as there is more than one navigation role on the page.